src/NUnit3TestAdapterInstall/NUnit3TestAdapterInstall.csproj

```diff
@@ -17,7 +14,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NUnit3TestAdapter</RootNamespace>
-    <AssemblyName>NUnit3TestAdapter</AssemblyName>
+    <TargetName>NUnit3TestAdapter</TargetName>
```

Comment:
NuGet restore gets confused when two projects have the same AssemblyName. Setting the TargetName to control the output .vsix name is more clear anyway since we're not even compiling an assembly in the VSIX project.